setup.cfg

```diff
@@ -91,6 +91,9 @@ ban-relative-imports = true
 python_files = test_*.py
 testpaths = test
 addopts = --verbose
+# 30 minutes should be enough for a single test to complete:
+timeout = 1800
+# timeout_method = thread does not work!
```

Comment:
looks like pytest-timeout will choose a good timeout_method for us, so this line could be dropped

Reply:
I think it always uses `signal` by default. I left that as a warning that everything breaks if we use the `thread` method and a timeout is actually triggered.